projects/storefrontapp-e2e-cypress/cypress/helpers/address-book.ts

```diff
@@ -50,6 +50,15 @@ export function accessPageAsAnonymous() {
   cy.location('pathname').should('contain', '/login');
 }
 
+export function accessAddressBookPage() {
```

Comment:
I think its better to use the cy.selectUserMenuOption command instead of manually doing it so that we don't need to repeat ourselves in the other e2e test and that since we're kind of 'refactoring', so I think we can make it more efficient and consistent.